compat/src/scheduler.js

```diff
@@ -18,7 +18,4 @@ export function unstable_runWithPriority(priority, callback) {
 	return callback();
 }
 
-export const unstable_now =
-	typeof performance === 'object' && typeof performance.now === 'function'
-		? performance.now.bind(performance)
-		: () => Date.now();
+export const unstable_now = () => performance.now();
```

Comment:
```suggestion
export const unstable_now = performance.now;
```

Reply:
heh - I checked because I'd hoped to do the same, but this throws an error due to the context being set to `globalThis`. I've changed it back to using bind.